examples/spawn-chain/README.md

```diff
@@ -37,3 +39,19 @@ cd www
 npm run start
 open http://localhost:8080
 ```
+
+### Get Demo Running on OS X
+
+If you don't already have the Lumen dev environment and just want to play with the demo, do these steps, ***then*** continue to the [link package](#link-package) steps.
+
+
+```
+brew uninstall rust
+brew install rustup
```

Comment:
Using brew to install `rustup` is a little weird.  It is supported, but weird as it leaves the first install of `rustup` managed by home brew instead of rustup itself.  I'd recommend using the instructions on the rustup site (https://rustup.rs/) instead.